packages/gestalt/src/Button.tsx

```diff
@@ -120,6 +120,23 @@ type Props = {
   type?: 'button' | 'submit';
 };
 
+function InternalButtonWrapper({ children }: { children: ReactNode }) {
+  const isInVRExperiment = useInExperiment({
+    webExperimentName: 'web_gestalt_visualRefresh',
+    mwebExperimentName: 'web_gestalt_visualRefresh',
+  });
+
+  return isInVRExperiment ? (
+    <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center', gap: 6 }}>
```

Comment:
any reason to not use Flex?

Reply:
oh, 6 px?

Reply:
ill bring this to the meeting on tuesday

Reply:
Fixed with the new Box solution!